addons/collapse-footer/addon.json

```diff
@@ -56,6 +56,22 @@
         "addonEnabled": ["infinite-scroll"]
       },
       "default": false
+    },
+    {
+      "name": "Toggle on",
+      "id": "mode",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "hover",
+          "name": "Hover"
+        },
+        {
+          "id": "click",
+          "name": "Click"
+        }
+      ],
+      "default": "hover"
```

Comment:
Should this setting be placed above `Only on infinite scrolling pages`?

Reply:
Done!